libs/auth/src/pages/cft-login/return.ts

```diff
@@ -0,0 +1,80 @@
+import type { Request, Response } from "express";
```

Comment:
This is a separate controller move that out into a new folder under pages called "cft-callback" or "cft-login-return" and keep the same format as other pages.